src/behavior-considered-undefined.md

```diff
@@ -26,8 +26,15 @@ code.
 * Evaluating a [dereference expression] (`*expr`) on a raw pointer that is
   [dangling] or unaligned, even in [place expression context]
   (e.g. `addr_of!(&*expr)`).
-* Breaking the [pointer aliasing rules]. `&mut T` and `&T` follow LLVM’s scoped
-  [noalias] model, except if the `&T` contains an [`UnsafeCell<U>`].
+* Breaking the [pointer aliasing rules]. `Box<T>`, `&mut T` and `&T` follow LLVM’s
+  scoped noalias model, except if the `&T` contains an [`UnsafeCell<U>`].
```

Comment:
I'm curious why remove the link to LLVM's docs, even though the sentence is still referring to "LLVM's scoped noalias model"? Without the link, how can a reader follow up on what that means?

If you do intend to remove the link to llvm's "noalias", can you also remove the link reference to http://llvm.org/docs/LangRef.html#noalias at the bottom of the page?

Reply:
Sorry, that was a mistake because this round-tripped through another document...